src/index.html

```diff
@@ -7,7 +8,78 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+
+      <p class="stars__star"></p>
+    </div>
```

Comment:
no needs to add an empty line between **_oneline_** siblings
```suggestion
    <div class="stars stars--0">
      <p class="stars__star"></p>
      <p class="stars__star"></p>
      <p class="stars__star"></p>
      <p class="stars__star"></p>
      <p class="stars__star"></p>
    </div>
```